src/ml_gym/frontend/dashboard/src/template/template.tsx

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import "./template.css";
+import StackedLineChartIcon from '@mui/icons-material/StackedLineChart';
+import TuneIcon from '@mui/icons-material/Tune';
+import ModelTrainingIcon from '@mui/icons-material/ModelTraining';
+import KeyboardDoubleArrowRightIcon from '@mui/icons-material/KeyboardDoubleArrowRight';
+import KeyboardDoubleArrowLeftIcon from '@mui/icons-material/KeyboardDoubleArrowLeft';
+import CellTowerIcon from '@mui/icons-material/CellTower';
+import { Link } from 'react-router-dom';
+import GlobalConfig from "../features/globalConfig/globalConfig";
+import { FilterConfigType } from "../app/datatypes"
+
+type TemplatePropsType = {
+    children?: React.ReactNode;
+    sideBarExpanded: boolean;
+    toggleSidebar: () => void;
+    setSelectedPageId: (page_id: number) => void;
+    selectedPageId: number
+    filterConfig: FilterConfigType
+    setFilterConfig: (filterConfig: FilterConfigType) => void
+};
+
+type SideBarPropsType = {
+    children?: React.ReactNode;
+    sideBarExpanded: boolean;
+    toggleSidebar: () => void;
+    setSelectedPageId: (page_id: number) => void;
+    selectedPageId: number
+};
+
+type ExpandCollapseButtonPropsType = {
+    sideBarExpanded: boolean;
+    toggleSidebar: () => void;
+};
+
+
+const ExpandCollapseButton: React.FC<ExpandCollapseButtonPropsType> = ({ sideBarExpanded, toggleSidebar }) => {
+    if (sideBarExpanded) {
+        return (<div className="nav_link expand-collapse-button"><KeyboardDoubleArrowLeftIcon className="expand-collapse-button" onClick={toggleSidebar} /></div>);
+    } else {
+        return (<div className="nav_link expand-collapse-button"><KeyboardDoubleArrowRightIcon onClick={toggleSidebar} /></div>);
+    }
+};
+
+
+
+const SideBar: React.FC<SideBarPropsType> = ({ toggleSidebar, sideBarExpanded, setSelectedPageId, selectedPageId }) => {
```

Comment:
Codacy has a fix for the issue: [Replace `·toggleSidebar,·sideBarExpanded,·setSelectedPageId,·selectedPageId·` with `⏎··toggleSidebar,⏎··sideBarExpanded,⏎··setSelectedPageId,⏎··selectedPageId,⏎`](https://app.codacy.com/gh/mlgym/mlgym/pullRequest?prid=9966992)
```suggestion
const SideBar: React.FC<SideBarPropsType> = ({
```